dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5316,18 +5262,6 @@ public String toString() {
               + "to commit results.")
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_FILE_PERSISTENCE_INITIAL_WAIT_TIME =
```

Comment:
This is the only user property key removed in this PR, because this PR intends to focus on master-only code and doesn't want any more complication. This piece here seems to rely on the master logic and doesn't make much sense staying, so I'm breaking the rule a little.